DOCSP-31602: cleanup for v1 (#52)
mongodb/docs-rust: source/fundamentals/connections/network-compression.txt

   driver selects the first one that is supported by the MongoDB
   instance that the driver is connected to.

This guide includes the following sections:

- :ref:`Add Compression Feature Flags <rust-nc-add-flags>`: describes
  how to add feature flags to your application for different compressors

- :ref:`Enable Network Compression <rust-nc-enable-compression>`:

>   describes how to implement network compression in your application by
>   using a ``Client`` instance

Reviewer: Suggestion:

I think this description introduces a new description "implement network compression in your application" which isn't similarly described on the page or in the section. I think the following could simplify the description without introducing one-time information:

```suggestion
  describes how to enable network compression on your ``Client`` instance
```
